channels/mx.m3u

```diff
@@ -65,3 +65,5 @@ https://5ca3e84a76d30.streamlock.net/gpacifico4/smil:mazatlan.smil/chunklist_w74
 https://5ca3e84a76d30.streamlock.net/gpacifico2/mochis.smil/chunklist_w52505888_b1428000_sleng.m3u8
 #EXTINF:-1 tvg-id="TVPobregon.mx" tvg-name="TVP obregon" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/mw4992J.png" group-title="",TVP obregon
 https://5ca3e84a76d30.streamlock.net/gpacifico3/obregon.smil/chunklist_w888631267_b1428000_sleng.m3u8
+#EXTINF:-1 tvg-id="MiamiTVLatino.mx" tvg-name="Miami TV Latino" tvg-country="MX" tvg-language="" tvg-logo="https://statics.yowi.tv/files/c74681cb7f75db303b818a931702600c44a9692d" group-title="XXX",Miami TV Latino
+https://59ec5453559f0.streamlock.net/canal18/smil:canal18/playlist.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/118852113-24390f80-b8db-11eb-8ad2-b5c2dba1ce88.png)

Reply:
I'm looking at it.